app/routes/games.js

```diff
@@ -1,6 +1,12 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  beforeModel: function () {
```

Comment:
same that we can potentially allow people watch games without logging in but this is cool for now